packages/rocketchat-livechat/package.js

```diff
@@ -61,18 +61,33 @@ Package.onUse(function(api) {
 	api.addFiles('client/collections/LivechatDepartment.js', 'client');
 	api.addFiles('client/collections/LivechatDepartmentAgents.js', 'client');
 	api.addFiles('client/collections/LivechatIntegration.js', 'client');
+	api.addFiles('client/collections/LivechatMonitoring.js', 'client');
 	api.addFiles('client/collections/LivechatPageVisited.js', 'client');
 	api.addFiles('client/collections/LivechatQueueUser.js', 'client');
 	api.addFiles('client/collections/LivechatTrigger.js', 'client');
 	api.addFiles('client/collections/LivechatInquiry.js', 'client');
 	api.addFiles('client/collections/livechatOfficeHour.js', 'client');
 	api.addFiles('client/collections/LivechatVisitor.js', 'client');
 
+	api.addFiles('client/lib/chartHandler.js', 'client');
+	api.addFiles('client/lib/dataHandler.js', 'client');
+	api.addFiles('client/lib/dateHandler.js', 'client');
```

Comment:
This one is duplicated, but as you've been using the `import` approach so you don't need any of them stated here.